MergerLogic/DataTypes/TMS.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using MergerLogic.Batching;
+using MergerLogic.Utils;
 
 namespace MergerLogic.DataTypes
 {
     public class TMS : HttpDataSource
     {
-        public TMS(DataType type, string path, int batchSize, Extent extent, int maxZoom, int minZoom = 0, bool isOneXOne = false,
+        public TMS(IUtilsFactory utilsFactory, IOneXOneConvetor oneXOneConvetor,
+            DataType type, string path, int batchSize, Extent extent, int maxZoom, int minZoom = 0, bool isOneXOne = false,
```

Comment:
Same here (and all HTTP classes), no need to pass DataType.

Reply:
fixed